gradle-config.gradle

```diff
@@ -0,0 +1,22 @@
+//
+// Configuration file for gradle build execution.
+//
+
+task accessToken {
+  def tokenFile = new File("${projectDir}/src/main/res/values/developer-config.xml")
+  if (!tokenFile.exists()) {
+    String tokenFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
+      "<resources>\n" +
+      "    <string name=\"mapbox_access_token\">" + "$System.env.MAPBOX_ACCESS_TOKEN" + "</string>\n" +
+      "</resources>"
+
+    if (tokenFileContents == null) {
```

Comment:
@Guardiola31337 great. could you amend this PR with the latest version of the script? Also, instead of failing the build with an `InvalidUserDataException`, would it make sense instead to show a warning and default to a string like `YOUR_MAPBOX_ACCESS_TOKEN_GOES_HERE`?